### PR TITLE
feat: add audit page title, subtitle, and CLI snippet

### LINKED ISSUE #3357: Add audit page header, subheader, and CLI snippet
As a user viewing the audit page, I would like to know what I'm viewing and where I am so I can use the app to its full potential and also communicate any errors to support with sufficient detail.

[Audit RFC
](https://www.notion.so/coderhq/v2-Enterprise-Audit-Logging-a6358d50a453413eae6441e689f023ac)
[Audit Mocks](https://www.figma.com/file/oy7echtkBVRuOCMBVSrr4T/Auditing?node-id=0%3A1) 

**Technical Considerations**
This ticket is blocked by #3356 

**AC**

- [x] In the new route page created in #3356 add a header and subheader
- [x] Add a tooltip to the subheader using our existing Tooltip component. The tooltip calls for a doc link - leave this out for now.
- [x] Add the CLI snippet as shown in the mocks. The CLI command may not be finalized; feel free to leave empty for now. Use the existing code snippet component we have in the FE. Code snippet should have a copy button.
- [x] Add tests and stories

**Definition of Done**
Above AC is completed.

### PR DESCRIPTION
**Pull request by Kira-Pilot**
resolves #3357 

We've begun to scaffold out the Audit Page. The CLI command widget may fluctuate in design down the road; right now, I just wanted to get something in there for MVP, so I'm using our existing `CodeExample` component. The CLI command is a placeholder rn as we figure out what we actually want. 

### Large Screens
Screen Shot 2022-08-08 at 3 22 12 PM

### Small Screens
Screen Shot 2022-08-08 at 3 54 22 PM

### CODE REVIEW

File: site/src/components/Tooltips/AuditHelpTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { FC } from "react"
+import { HelpTooltip, HelpTooltipText, HelpTooltipTitle } from "./HelpTooltip"
+
+export const Language = {
+  title: "What is an audit log?",
+  body: "An audit log is a record of events and changes made throughout a system.",
+}
+
+export const AuditHelpTooltip: FC = () => {
```
Comment (presleyp) [inline site/src/components/Tooltips/AuditHelpTooltip.tsx:9]: Good use of component nesting here (tooltips)

**Review by jsjoeio — APPROVED** [review]
Nice work!

**Review by presleyp — APPROVED** [review]
Nice!

**Comment by presleyp** [comment]
The small viewport story doesn't have a small viewport for me, do I need to configure anything?

**Comment by Kira-Pilot** [comment]
@presleyp hmmm, are you seeing something weird in Chromatic CI (if so, can you post a screenshot?) or do you mean you're having trouble when running locally? The Chromatic CI snapshot looks right to me. In a smaller viewport, the code block containing the CI command can be found under the page title and subtitle as opposed to across from it.

**Comment by presleyp** [comment]
I meant Chromatic CI but actually now it looks fine!

**Pull request merged**